src/tiled/changepolygon.cpp

```diff
@@ -76,3 +76,50 @@ void TogglePolygonPolyline::toggle()
 {
     mMapObject->setShape((mMapObject->shape() == MapObject::Polygon) ? MapObject::Polyline : MapObject::Polygon);
 }
+
+SplitPolyline::SplitPolyline(MapDocument *mapDocument,
+                             MapObject *mapObject,
+                             int index)
+    : mMapDocument(mapDocument)
+    , mFirstPolyline(mapObject)
+    , mEdgeIndex(index)
+    , mObjectIndex(-1)
+{
+    mSecondPolyline = mFirstPolyline->clone();
+    mSecondPolyline->resetId();
+
+    setText(QCoreApplication::translate("Undo Commands", "Split Polyline"));
+}
+
+void SplitPolyline::undo()
+{
+    mObjectIndex = mMapDocument->mapObjectModel()->removeObject(mFirstPolyline->objectGroup(),
+                                                                mSecondPolyline);
+
+    QPolygonF firstPolygon = mFirstPolyline->polygon();
+    QPolygonF secondPolygon = mSecondPolyline->polygon();
+
+    firstPolygon.append(secondPolygon);
```

Comment:
Because it's compiling against Qt 5.4, which does not have a `QVector::append` overload that will append a vector. You can use the `+=` operator instead:
```c++
firstPolygon += secondPolygon;
```